src/components/views/dialogs/DevtoolsDialog.tsx

```diff
@@ -578,32 +628,41 @@ class AccountDataExplorer extends React.PureComponent {
                     { rows }
                 </FilteredList>
             </div>
-            <div className="mx_Dialog_buttons">
+            <div className="mx_Dialogbuttons">
                 <button onClick={this.onBack}>{ _t('Back') }</button>
-                { !this.state.message && <div style={{float: "right"}}>
-                    <input id="isRoomAccountData" className="mx_DevTools_tgl mx_DevTools_tgl-flip" type="checkbox" onChange={this._onChange} checked={this.state.isRoomAccountData} />
-                    <label className="mx_DevTools_tgl-btn" data-tg-off="Account Data" data-tg-on="Room Data" htmlFor="isRoomAccountData" />
-                </div> }
+                <div style={{float: "right"}}>
+                    <input id="isRoomAccountData" className="mx_DevTools_tgl mx_DevTools_tgl-flip"
+                        type="checkbox"
+                        checked={this.state.isRoomAccountData}
+                        onChange={this.onChange}
+                    />
+                    <label className="mx_DevTools_tgl-btn"
+                        data-tg-off="Account Data"
+                        data-tg-on="Room Data"
+                        htmlFor="isRoomAccountData"
+                    />
+                </div>
             </div>
         </div>;
     }
 }
 
-class ServersInRoomList extends React.PureComponent {
-    static getLabel() { return _t('View Servers in Room'); }
+interface IServersInRoomListState {
+    query: string;
+}
 
-    static propTypes = {
-        onBack: PropTypes.func.isRequired,
-        room: PropTypes.instanceOf(Room).isRequired,
-    };
+class ServersInRoomList extends React.PureComponent<IExplorerProps, IServersInRoomListState> {
+    static getLabel() { return _t('View Servers in Room'); }
 
     static contextType = MatrixClientContext;
 
+    private servers: React.ReactElement[];
+
     constructor(props) {
         super(props);
 
         const room = this.props.room;
-        const servers = new Set();
+        const servers: Set<string> = new Set();
```

Comment:
alternatively:
```suggestion
        const servers = new Set<string>();
```

Reply:
Ah thanks, indeed that's slightly shorter, fixed.